addons/analytic/tests/common.py

```diff
@@ -41,3 +42,23 @@ def setup_independent_company(cls, **kwargs):
     def get_default_groups(cls):
         groups = super().get_default_groups()
         return groups + cls.env.ref('analytic.group_analytic_accounting')
+
+    @classmethod
+    def setup_independent_user(cls):
+        default_groups = cls.env.ref('base.group_system') + cls.get_default_groups()
+        """ Removes access rights linked to timesheet and project as these add
```

Comment:
Feels weird to have that as docstring here.  I would say you could put it as first line of method, or use `#`